openid-provider-commands-1_0.md

```diff
@@ -92,23 +92,82 @@ This specification defines the following terms:
 
 This specification defines a Command Request containing a Command Token sent from the OP to the RP, and a Command Response returned from the RP to the OP.
 
-```
+```bash
 +------+  Command request       +------+
 |      |---- Command Token ---->|      |
 |  OP  |                        |  RP  | 
 |      |<-----------------------|      |
 +------+      Command response  +------+
 ```
 
-## Command Usage Overview
+The OP may provide a callback endpoint and a callback token for the RP to request a command be sent by the OP such as a metadata or audit_tenant command, or to send the results of an asynchronous command. 
```

Comment:
Why is this paragraph added in this PR? I didn't think this PR was adding anything new around callback behavior